packages/slice-machine/src/components/VideoPopover/VideoPopover.css.ts

```diff
@@ -93,7 +93,11 @@ export const videoButton = style([
 
 export const closeButton = style([
   block,
-  sprinkles({ color: colors.purple12 }),
+  sprinkles({
+    color: colors.purple12,
+    padding: 6,
```

Comment:
Not sure to understand, I see 4px as global padding and then 6px here so 10px before the icon, is that what we want?

Reply:
Depend which hover-card you look at: this one says 6 https://www.figma.com/file/wb7ujCJ4j7EiP7aDDGRYXB/SliceMachine%E2%84%A2-UI?node-id=4018%3A320396&mode=dev

but I can change it back to four and it shouldn't make much difference...

Reply:
Oh no that's perfect your link! Was not on that one, thank you!